api/prometheus/v1/api_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/api"
+	json "github.com/json-iterator/go"
```

Comment:
This line has to be before L28 to satisfy the CI style test.

However, for consistency, the named import should be in its own block (separated by a blank line), and L28 should be in the same block as L31 and L32.

Reply:
If we're going to be this picky (which I think is fine) I'd suggest adding a `make imports` function similar to https://github.com/jacksontj/promxy/blob/master/Makefile#L17